modules/mobile/spec/request/v1/immunizations_request_spec.rb

```diff
@@ -168,46 +130,57 @@
 
       context 'when an immunization group name is COVID-19 and there is a manufacturer provided' do
         it 'uses the vaccine manufacturer in the response' do
-          immunizations_request_covid_paginated
-          expect(response.parsed_body['data'][0]['attributes']).to eq(
-            {
-              'cvxCode' => 207,
-              'date' => '2020-12-18T12:24:55Z',
-              'doseNumber' => nil,
-              'doseSeries' => nil,
+          VCR.use_cassette('mobile/lighthouse_health/get_immunizations', match_requests_on: %i[method uri]) do
+            get '/mobile/v1/health/immunizations', headers: sis_headers, params: { page: { size: 14, number: 1 } }
+          end
+          covid_with_manufacturer_immunization = response.parsed_body['data'].select do |i|
+            i['id'] == 'I2-R5T5WZ3D6UNCTRUASZ6N6IIVXM000000'
+          end
+
+          expect(covid_with_manufacturer_immunization.dig(0, 'attributes')).to eq(
+            { 'cvxCode' => 213,
+              'date' => '2021-04-18T09:59:25Z',
+              'doseNumber' => 'Series 1',
+              'doseSeries' => 'Series 1',
               'groupName' => 'COVID-19',
-              'manufacturer' => 'Moderna US, Inc.',
-              'note' =>
-                'Dose #1 of 2 of COVID-19, mRNA, LNP-S, PF, 100 mcg/ 0.5 mL dose vaccine administered.',
-              'reaction' => nil,
-              'shortDescription' => 'COVID-19, mRNA, LNP-S, PF, 100 mcg or 50 mcg dose'
-            }
+              'manufacturer' => 'TEST MANUFACTURER',
+              'note' => 'Sample Immunization Note.',
+              'reaction' => 'Other',
+              'shortDescription' => 'SARS-COV-2 (COVID-19) vaccine, mRNA, spike protein, LNP, preservative free, 30' \
+                                    ' mcg/0.3mL dose' }
           )
         end
       end
 
       context 'when an immunization group name is COVID-19 and there is no manufacturer provided' do
         it 'sets manufacturer to nil' do
-          immunizations_request_covid_no_manufacturer_paginated
-          expect(response.parsed_body['data'][0]['attributes']).to eq(
-            {
-              'cvxCode' => 207,
-              'date' => '2021-01-14T09:30:21Z',
-              'doseNumber' => nil,
-              'doseSeries' => nil,
+          VCR.use_cassette('mobile/lighthouse_health/get_immunizations', match_requests_on: %i[method uri]) do
+            get '/mobile/v1/health/immunizations', headers: sis_headers, params: { page: { size: 14, number: 1 } }
+          end
+
+          covid_no_manufacturer_immunization = response.parsed_body['data'].select do |i|
+            i['id'] == 'I2-LJAZCGMN3BZVQVKQCVL7KMTHJA000000'
+          end
```

Comment:
This is what was causing the issue. for some reason, on github trying to get the exact page with size of one was causing an authentication error. I'm pretty confident this is a CI bug, not a bug in the code.